test/e2e/nft/remove-nft.spec.js

```diff
@@ -19,27 +19,18 @@ describe('Remove NFT', function () {
     await withFixtures(
       {
         dapp: true,
-        fixtures: new FixtureBuilder().build(),
+        fixtures: new FixtureBuilder().withNftControllerERC721().build(),
         ganacheOptions,
         smartContract,
         title: this.test.title,
       },
-      async ({ driver, _, contractRegistry }) => {
-        const contractAddress =
-          contractRegistry.getContractAddress(smartContract);
+      async ({ driver }) => {
         await driver.navigate();
         await driver.fill('#password', 'correct horse battery staple');
         await driver.press('#password', driver.Key.ENTER);
 
-        // After login, go to NFTs tab and import an NFT
-        await driver.clickElement('[data-testid="home__nfts-tab"]');
-        await driver.clickElement({ text: 'Import NFTs', tag: 'a' });
-
-        await driver.fill('[data-testid="address"]', contractAddress);
-        await driver.fill('[data-testid="token-id"]', '1');
-        await driver.clickElement({ text: 'Add', tag: 'button' });
-
         // Open the details and click remove nft button
```

Comment:
We now import the ERC721 using the fixture. We have separate tests for the import flow